src/js/util.js

```diff
@@ -96,8 +96,8 @@ var Util;
             return !!(obj && obj.nodeType === 1);
         },
 
-        now: function now() {
-            return Date.now() || new Date().getTime();
+        now: Date.now || function now() {
+            return +new Date();
```

Comment:
Thanks for finding and fixing this.  I think you were right when you said we can just use `Date.now()` safely in all our browsers, this was copy and pasted from underscore js so we were probably being safer than we needed to be.